.github/workflows/ocl-subscription-module.yml

```diff
@@ -39,4 +40,4 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: Screen recordings of failed tests
-          path: cypress/videos/oclsubscriptionmodule
+          path: qaframework-bdd-tests/cypress/videos/oclsubscriptionmodule
```

Comment:
The video that gets uploaded here is pretty much useless. Would it also be possible to attach the screenshots of the failed tests?

Reply:
> Would it also be possible to attach the screenshots of the failed tests?

The build plan summery provided by git-hub actions is good enough to show which tests have failed and those that have passed.https://github.com/openmrs/openmrs-contrib-qaframework/runs/5355693711?check_suite_focus=true#step:7:44

Reply:
Yes, it shows which tests failed. What it doesn't show is *why* those tests failed. I can't tell which step in the flow causes the error: "CypressError: `cy.type()` cannot accept an empty string. You need to actually type something." nor can I see which field it's trying to type in. A screenshot would solve both of those problems (the video doesn't actually capture anything useful).

Reply:
![Import Dictionary via subscription module -- The User should be able to install the dictionary in openmrs (failed)](https://user-images.githubusercontent.com/38831673/156974793-e3dcbf6b-01f3-4097-926d-ec537948f912.png)

@ibacher the credentials `OCL_USER_NAME`, `OCL_USER_PASSWORD,` `OPENMRS_USER_NAME`, `OPENMRS_USER_PASSWORD` and `OCL_USER_TOKEN `are failing to be imported into the cypress steps definition hence CypressError: `cy.type()`

Have instead used the credentials directly in the steps file( though its not good to expose these credentials in steps definition). I know these credentials were set as secret values however, could there be a possibility that these secrete credentials are expired and need to be updated?